cross/toolchain.cmake

```diff
@@ -79,8 +86,10 @@ add_compile_param(CMAKE_MODULE_LINKER_FLAGS "${CROSS_LINK_FLAGS}" "TOOLCHAIN_EXE
 
 # Specify compile options
 add_compile_options("--sysroot=${CROSS_ROOTFS}")
+if (("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang") OR ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC"))
```

Comment:
nit, from the existing usages of `STREQUAL`, looks like the simplified expression works as well for string comparisons (here and other places):

```diff
- if (("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang") OR ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC"))
+ if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang" OR CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
```

Reply:
Some folks are using this kind of condition:

```cmake
if(CMAKE_CXX_COMPILER_ID MATCHES "(Clang|MSVC)")
```

maybe that will fix, as the ID might not be exactly `Clang` in rootfs?

Reply:
I think we want to have a consistent style for compiler detection across the code. I'll keep the current change as it is since I got rid of MSVC as well.

Reply:
done